### PR TITLE
Nx migrate latest

### PR DESCRIPTION
**Pull request by gabalafou**

### CODE REVIEW

**Comment by bskinn** [comment]
Any specific review actions here, @gabalafou, other than a scan across both built sites?

**Comment by bskinn** [comment]
@gabalafou we could perhaps rollup in this PR:

- #559
- #506 
- #486

**Review by bskinn — COMMENTED** [review]
Another possibility for rollup

File: package.json [hunk 1]
```diff
@@ -59,53 +60,55 @@
...
+    "nx": "15.9.2",
     "nx-stylelint": "^13.2.1",
-    "postcss": "^8.4.14",
-    "prettier": "^2.5.1",
-    "react-test-renderer": "17.0.2",
-    "sass": "1.49.9",
+    "postcss": "8.4.19",
+    "prettier": "2.8.7",
+    "react-test-renderer": "18.2.0",
+    "sass": "1.55.0",
```
[the remark this answers is not in the document]
Comment (bskinn) [reply package.json:105]: Possibly moot given #486

[the remark this answers is not in the document]
Comment (bskinn) [reply package.json:105]: #646 now obsoleted by #732

**Comment by gabalafou** [comment]
I think too much has happened on the `develop` branch since I opened this PR. I'm going to start over from scratch, using this PR as a guide rather than the foundation.

**Pull request closed without merging**